var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -38,7 +38,11 @@ class Scotch(CMakePackage, MakefilePackage):
 
     build_system(conditional("cmake", when="@7:"), "makefile", default="cmake")
     variant("threads", default=True, description="use POSIX Pthreads within Scotch and PT-Scotch")
-    variant("mpi_thread", default=False, description="use multi-threaded algorithms")
+    variant(
+        "mpi_thread",
+        default=True,
```

Comment:
Is there a reason why this is changed to `True`?

Reply:
That's odd, I think it was changed by mistake. Just reverted it.